### DISCUSSION TITLE
My map is displayed too small

### DISCUSSION THREAD

**Discussion by SonyLawTop** [Live-Charts/LiveCharts2#1615] [Q&A]
Hi, I'm having trouble with a GeoMap I created. The map appears too small, and I would like to know how to apply a "zoom to fit" so that the map fits the screen properly. I have attached both a screenshot and the corresponding JSON file for reference.
[jateng.json](https://github.com/user-attachments/files/17203338/jateng.json)
[image](https://github.com/user-attachments/assets/5494c384-019a-4886-847d-ec5274b3a3eb)

```
private void loadGeoMap()
{
    StreamReader sr = new StreamReader(new MemoryStream(Properties.Resources.jateng));
    string jsonData = sr.ReadToEnd();
    geoMap1.ActiveMap = Maps.GetMapFromStreamReader<SkiaSharpDrawingContext>(new StreamReader(new MemoryStream(Encoding.UTF8.GetBytes(jsonData))));
}
```

**Comment by beto-rodriguez** [comment]
I created an issue based on this, I am not sure if this feature is supported now, if not, it should be.

**Reply by SonyLawTop** [reply]
Thank you for creating the issue! I appreciate it.
I hope the "zoom to fit" feature will be available soon, as it would be incredibly helpful.
